auditor.md

```diff
@@ -0,0 +1,21 @@
+## ArbitrablePermissionList
+ArbitrablePermissionList implements an arbitrator curated registry.
+
+### Notes
+- Anyone can post an item with a deposit.
+- If no one complains within a defined time period, the item is added to the registry.
+- Someone can complain and also post a deposit. If someone does, a dispute is created. The winner of the dispute gets the deposit of the other party and the item is added or removed accordingly.
+- During the time of the dispute, the item is shown as blacklisted unless it already won a previous dispute.
```

Comment:
This part was specific to blacklists. Comments on the file has been changed accordingly.